implementations/rust/ockam/ockam_transport_tcp/src/router/mod.rs

```diff
@@ -120,13 +120,6 @@ impl Worker for TcpRouter {
     type Context = Context;
     type Message = RouterMessage;
 
-    async fn initialize(&mut self, ctx: &mut Context) -> Result<()> {
```

Comment:
@spacekookie FYI. This was causing a race condition. Are there other places like this?